src/Validator/index.js

```diff
@@ -142,3 +142,11 @@ export const validateDarkTheme = (parameter?: any): boolean => {
 
   return window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
```suggestion
  return typeof window !== 'undefined' && window.matchMedia('(prefers-color-scheme: dark)').matches
```

I think we can add here a SSR check too